src/app/header/components/selector/selector.component.scss

```diff
@@ -0,0 +1,19 @@
+.options {
+    inset: 0px 0px auto 0px !important;
+}
+.btn {
+    outline: 2px solid #13C1AC;
+    color: #455D63;
+    &:hover{
+        color:white;
+        background-color: #13C1AC;
```

Comment:
You could be using variables for the colors :D

Reply:
Created a variable for the color!

Reply:
But you're not using it here!